### PR TITLE
app: enable OUTPUT_DISASSEMBLY for Zephyr builds

### PR DESCRIPTION
**Pull request by kv2019i**
With Zephyr commit cc5763344709 ("Build system: disable `OUTPUT_DISASSEMBLY` by default"), generation of zephyr.lst is disabled by default.

Enable disassembly generation in SOF configuration as it is expected by SOF builds rules for reproducible builds.

### CODE REVIEW

File: app/prj.conf [hunk 1]
```diff
@@ -17,6 +17,9 @@ CONFIG_BUILD_OUTPUT_STRIPPED=y
 # default value.
 CONFIG_CLEANUP_INTERMEDIATE_FILES=n
 
+# Generate zephyr.lst file
+CONFIG_OUTPUT_DISASSEMBLY=y
```
Comment (btian1) [inline app/prj.conf:21]: what's the benefit of output lst file during build? who will use it? it will somehow make build slower.

**Review by btian1 — APPROVED** [review]

**Review by aiChaoSONG — APPROVED** [review]

File: app/prj.conf (hunk 1 above)
Comment (aiChaoSONG) [reply app/prj.conf:21]: xtensa-build-zephyr.py will package this file to a compressed file for debug, this file is very useful for debug.  It seems without this file, xtensa-build-zephyr.py will complain.

**Comment by kv2019i** [comment]
Zephyr mainline tests pass again, proceeding with merge.

**Comment by greg-intel** [comment]
I ran the zephyr build job, it had been failing in two to three minutes, and has now passed the five minute mark, so it looks like this PR has fixed the issue, thanks!

**Comment by btian1** [comment]
image

will it fix above CI issue?

**Pull request merged**